NOTICE

```diff
@@ -0,0 +1,14 @@
+ENSIME Server
+Copyright 2010 - 2016 Aemon Cannon
+Copyright 2014 - 2017 Sam Halliday
+Copyright 2010 - 2017 https://github.com/ensime/ensime-server/graphs
```

Comment:
@sugakandrey you'll be a major committer once graph branch lands, you should add your name here :smile: